scripts/vnet_route_check.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+import json
+from swsssdk import ConfigDBConnector
+from swsssdk import SonicV2Connector
+
+
+def get_vnet_intfs():
+    config_db = ConfigDBConnector()
```

Comment:
general comments for all function: please add function description and what it is expected to return and what is the